tracer/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "tracer"
```

Comment:
It'd be nice to elaborate in the commit message the purpose of this tracer and how it will be used.